api/maven-api-model/src/main/mdo/maven.mdo

```diff
@@ -1945,6 +1965,178 @@
+            The generated build system will compile the sources from this directory using the Java compiler when the
+            project is built. The path given in the {@code directory} field is relative to the project descriptor.
+            The default directory for the main Java sources is {@code "src/main/java"}.</p>
+
+            <p>The <b>resources</b> language is used for specifying a directory containing the class-path
+            or module-path resources such as properties files or scripts associated with a project.
+            This directory is meant to be different from the main source directory,
+            in that its contents will be copied to the output directory in most cases
+            (since scripts are interpreted rather than compiled).
+            The default directory for the main resources is {@code "src/main/resources"}.</p>
+
+            <p>If no language is specified, the default is {@code java}.</p>
+            ]]>
+          </description>
+          <type>String</type>
+          <defaultValue>main</defaultValue>
+        </field>
+        <field>
+          <name>module</name>
+          <version>4.1.0+</version>
+          <description>
+            <![CDATA[
+            Name of the Java module (or other language-specific module) which is built by the sources.
+            This element can be specified in a Maven project containing multiple Java modules.
+            It is generally not needed for non-modular projects, or for modular projects having only one module.
+
+            <p>If a module name is specified for resources or script files,
+            then this value modifies the directory where the files will be copied.
+            For example, if a Java module name is "foo.biz", then the {@code foo/bar.properties}
+            resource file will be copied as {@code foo.biz/foo/bar.properties}.</p>
+
+            <p>This element can be combined with the {@code targetVersion} element for specifying sources,
+            scripts or resources that are specific to both a particular module and a target version.</p>
+            ]]>
+          </description>
+          <type>String</type>
+        </field>
+        <field>
+          <name>targetVersion</name>
+          <version>4.1.0+</version>
+          <description>
+            <![CDATA[
+            The version of the platform where the code will be executed.
+            In a Java environment, this is the value of the {@code --release} compiler option.
+            If a Java project contains multiple main sources with different target versions,
+            then a multi-version <abbr>JAR</abbr> file will be created.
+            If this element is omitted, then the default target version is the compiler default value,
+            which is usually the version of the Java environment running Maven.
+
+            <p>If a target version is specified for resources or script files,
```

Comment:
Should that be `if a target version, different from the default target, version is specified` ?

Reply:
Agreed. So specifying a target version for the `source` is not sufficient to create a multi-release jar. That's why I think the sentence:
> If a target version is specified for resources or script files, then this value modifies the directory where the files will be copied.

is not completely accurate.  This is only the case if the version is not the base one.

Reply:
Ah, understood. You are right. I will fix tomorrow.